src/test/optimizer/dp_ccp_test.cpp

```diff
@@ -86,15 +86,16 @@ TEST_F(DpCcpTest, JoinOrdering) {
 
   // clang-format off
   const auto expected_lqp =
-  PredicateNode::make(equals_(b_a, c_a),
-    JoinNode::make(JoinMode::Inner, equals_(a_a, c_a),
+    JoinNode::make(JoinMode::Inner, std::vector<std::shared_ptr<AbstractExpression>>{
```

Comment:
just fyi, you could use the `expression_vector` utility instead of `std::vector<std::shared_ptr<AbstractExpression>>`